app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsFeedClient.kt

```diff
@@ -0,0 +1,62 @@
+package org.wikipedia.feed.suggestededits
+
+import android.content.Context
+import io.reactivex.android.schedulers.AndroidSchedulers
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.schedulers.Schedulers
+import org.apache.commons.lang3.StringUtils
+import org.wikipedia.WikipediaApp
+import org.wikipedia.dataclient.WikiSite
+import org.wikipedia.dataclient.restbase.page.RbPageSummary
+import org.wikipedia.editactionfeed.provider.MissingDescriptionProvider
+import org.wikipedia.feed.FeedCoordinator
+import org.wikipedia.feed.dataclient.FeedClient
+import org.wikipedia.feed.model.Card
+import org.wikipedia.page.PageTitle
+
+class SuggestedEditsFeedClient(var translation: Boolean) : FeedClient {
+    private val disposables = CompositeDisposable()
+    var sourceDescription: String = ""
+    var targetPageTitle: PageTitle? = null
+    private val app = WikipediaApp.getInstance()
+    var summary: RbPageSummary? = null
+    override fun request(context: Context, wiki: WikiSite, age: Int, cb: FeedClient.Callback) {
+        cancel()
+        getArticleWithMissingDescription(cb, wiki)
+    }
+
+    private fun getArticleWithMissingDescription(cb: FeedClient.Callback, wiki: WikiSite) {
+        if (translation) {
+            disposables.add(MissingDescriptionProvider.getNextArticleWithMissingDescription(WikiSite.forLanguageCode(app.language().appLanguageCodes.get(0)), app.language().appLanguageCodes.get(1), true)
```

Comment:
Use `app.language().appLanguageCodes[0]` and `app.language().appLanguageCodes[1]`

Reply:
Done.